compiler/luci-interpreter/src/kernels/CMakeLists.txt

```diff
@@ -63,14 +63,16 @@ set(SOURCES
     TransposeConv.h
     TransposeConv.cpp
     Unpack.h
-    Unpack.cpp)
+    Unpack.cpp
+    ${TensorFlowSource_DIR}/tensorflow/lite/kernels/internal/quantization_util.cc)
```

Comment:
Please move this file to the list below (line 69).

Reply:
Okay, I will fix it.